### PR TITLE
Commentcounter tester

### PR DESCRIPTION
**Pull request by MohamedCK**
- Create a comment counter.
- Create a test for the length of the comment.

### CODE REVIEW

File: coverage/lcov-report/sorter.js [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+        }
+        cols = loadColumns();
+        loadData();
+        addSearchBox();
+        addSortIndicators();
+        enableUI();
+    };
+})();
+
+window.addEventListener('load', addSorting);
```
Comment (tinatsina) [inline coverage/lcov-report/sorter.js:196]: ## Linter errors ❌ ❌ ❌ 
- you have two Linter errors
- Stylelint
- ESlint
- fix these errors before I can review
- 
Jimmy Fallon Lets Go

**Review by Suzi216 — APPROVED** [review]
Hi @MohamedCK , 

## Status: APPROVED 🥳 🥳 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- Linters are passing ✔️ 
- Great job with your design and functionalities 👏 

## Optional suggestions
- Please fix the linters

File: coverage/lcov-report/sorter.js (hunk 1 above)
Comment (MohamedCK) [reply coverage/lcov-report/sorter.js:196]: Come on I was about to fix @tinatsina

**Review by tinatsina — APPROVED** [review]
## Hi @MohamedCK  🙋‍♂️, 

Good job partner 🤝 🤝 🤝 

Scar Precisely

## Highlights
- No Linter errors ✅
- Well file structure ✅
- Good summary description ✅

Cheers and Happy coding!👏👏👏

------
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Pull request merged**